samples/kvsWebRTCClientViewer.c

```diff
@@ -165,6 +195,26 @@ INT32 main(INT32 argc, CHAR* argv[])
         goto CleanUp;
     }
 
+    PRtcDataChannel pDataChannel = NULL;
```

Comment:
Encapsulate this entire section this way:

```
#ifdef ENABLE_DATA_CHANNEL
<code block>
#endif

Checkout usage [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/c4a13309b6559d19529ca8049a50d4e2b241564f/src/source/PeerConnection/PeerConnection.c#L143)

Reply:
Added ENABLE_DATA_CHANNEL encapsulation here